src/main/java/org/jenkinsci/plugins/workflow/flow/FlowDurabilityHint.java

```diff
@@ -33,14 +33,11 @@
  * @author Sam Van Oort
  */
 public enum FlowDurabilityHint {
-    PERFORMANCE_OPTIMIZED(false, false, "Performance-optimized: much faster (requires clean shutdown to save running pipelines)",
-            "Avoids writing data with every step, avoids atomic writes of data.  Pipelines can resume if Jenkins shuts down cleanly, but running pipelines lose step information and can't resume if Jenkins unexpectedly fails."),
+    PERFORMANCE_OPTIMIZED(false, false, Messages.FlowDurabilityHint_PERFORMANCE_OPTIMIZED_description(), Messages.FlowDurabilityHint_PERFORMANCE_OPTIMIZED_tooltip()),
```

Comment:
These should be `Localizable`s: `Messages._FlowDurabilityHint_PERFORMANCE_OPTIMIZED_description()`, etc.

Reply:
@jglick  Huh?  They are Localizables, using Messages.xxxx

Reply:
Nope, you did not see or understand my earlier review comment I guess—these fields in the enum should be of type `Localizable` not `String`, rendered if and when displayed, which is why I said `Messages._FlowDurabilityHint_PERFORMANCE_OPTIMIZED_description()` not `Messages.FlowDurabilityHint_PERFORMANCE_OPTIMIZED_description()`.

Anyway, not important to change now, as it can be fixed later without breaking API compatibility.

Reply:
Ah, I see, I missed your underscore -- yeah, it's a subtle difference (this one is localized to the master, the other might be localized differently). 

Definitely worth a fix later though.